src/LayerPanel/LayerPanelLayersPage/index.js

```diff
@@ -273,8 +277,15 @@ class LayerPanelLayersPage extends Component {
 
     if (layerCheckboxClick && layerVisibility === INDETERMINATE) {
       layer.setVisible(true)
+      layer.set('_ol_kit_layerpanel_visibility', true)
+    } else if (layerCheckboxClick) {
+      const lv = !layer.getVisible()
```

Comment:
A better name might be `toggledLayerVisibility` (assuming we're trying not to shadow/change the in-scope `layerVisibility` ... maybe it just shouldn't be `const` so we're not introducing another variable for no particular need?).